src/pudl/validate.py

```diff
@@ -624,30 +515,6 @@ def plot_vs_bounds(df, validation_cases):
         bounds_histogram(df, **args)
 
 
-def plot_vs_self(df, validation_cases):
-    """Validate a bunch of distributions against themselves."""
-    for args in validation_cases:
-        try:
-            vs_self(df, **args)
-        except ValueError as e:
-            logger.error(str(e))
-            warnings.warn("ERROR: Validation Failed", stacklevel=1)
-
-        historical_histogram(df, test_df=None, **args)
```

Comment:
with these changes it looks like historical_histogram isn't called anywhere else, so we can get rid of it, too

Reply:
You know, I removed this initially, and then I put it back because it seems like we'll need to debug some failing `vs_bounds` tests or better understand the historical annual variability in the distributions that we're checking in order to select bounds that can pass in both the full ETL and fast ETL scenarios, and it seemed like it would be useful in that context.